src/hooks/usePersonalizedFeatureCards.ts

```diff
@@ -1,50 +1,62 @@
-import type { FeatureCardData } from '@/types/strapi';
 import { useAccount } from '@lifi/wallet-management';
 import { useQuery } from '@tanstack/react-query';
+import type { FeatureCardData } from '@/types/strapi';
 
 export interface UsePersonalizedFeatureCardsProps {
-  featureCards: FeatureCardData[] | undefined;
+  data: FeatureCardData[];
   isSuccess: boolean;
   isConnected: boolean;
 }
 
-const STRAPI_CONTENT_TYPE = 'jumper-users';
+function getFeatureCardsEndpoint(walletAddress: string): string {
+  return `${process.env.NEXT_PUBLIC_BACKEND_URL}/wallets/${walletAddress}/feature-cards`;
+}
+
+const STRAPI_CONTENT_TYPE = 'feature-cards';
 export const usePersonalizedFeatureCards =
   (): UsePersonalizedFeatureCardsProps => {
     const { account } = useAccount();
 
+    const { data: fcCardData, isSuccess: fcCardDataIsSuccess } = useQuery({
+      queryKey: ['jumperUser', account?.address],
+      queryFn: async () => {
+        const response = await fetch(
+          getFeatureCardsEndpoint(account?.address!),
+        );
+        const result = await response.json();
```

Comment:
i think there should be some `try/catch` or check if `response.ok` in case backend throws an unexpected error